tools/cmake/Makefile

```diff
@@ -29,7 +28,20 @@ HOST_CONFIGURE_VARS += \
 
 HOST_CONFIGURE_ARGS := \
 	$(if $(MAKE_JOBSERVER),--parallel="$(MAKE_JOBSERVER)") \
-	--prefix=$(STAGING_DIR_HOST)
+	--prefix=$(STAGING_DIR_HOST) \
+	--generator=Ninja
```

Comment:
This seems like a pretty big step which should be reflected in the commit subject (not just commit message).

Reply:
Done.